package.json

```diff
@@ -2300,6 +2301,7 @@
         "node-gyp-build": "^4.2.3",
         "node-has-native-dependencies": "^1.0.2",
         "node-html-parser": "^1.1.13",
+        "npm-force-resolutions": "0.0.10",
```

Comment:
This package allows us to override the child packages to point to specific versions.